app/Http/Controllers/PrintController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\User;
+use App\PrintAccount;
+
+class PrintController extends Controller
+{
+    public function index() {
+        return view('print.app');
+    }
+
+    public function modify_balance(Request $request) {
+        $printAccount = User::find($request->user_id)->printAccount;
```

Comment:
Use [findOrFail ](https://laravel.com/docs/5.8/eloquent#retrieving-single-models) to handle invalid ids.

Reply:
Done.